projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1522,16 +1571,17 @@ public static function jetpack_connection_test() {
 	 * @return bool
 	 */
 	public static function view_jetpack_connection_test_check() {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
```

Comment:
I'd like us to get into the practice of explaining why when we ignore nonce checks.
```suggestion
		// phpcs:disable WordPress.Security.NonceVerification.Recommended -- This is verifying the trusted caller via a shared private key and timestamp.
```